include/rmm/mr/device/fixed_multisize_memory_resource.hpp

```diff
@@ -39,13 +39,10 @@
 using cudaStream_t = struct CUstream_st*;
 
 namespace rmm {
-
 namespace mr {
-
-namespace {
-
+namespace detail {
```

Comment:
Anonymous namespace shouldn't be used in headers.

Reply:
TIL https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file